src/components/Auth/api.ts

```diff
@@ -87,22 +87,46 @@ export default class AuthApi extends RestApi {
     return this.fetchUser(json.localId);
   }
 
-  async updateConfig(
+  async updateAllowDuplicateEmails(
```

Comment:
I'd actually suggest we should keep `updateConfig` as a single method here since the logic for both `allowDuplicateEmails` and `usageMode` is largely the same and it's one API endpoint after all. We can just make the parameter type `{ signIn?: { allowDuplicateEmails?: boolean }, usageMode?: UsageModes }` so callers can update whichever they like. The return type can be non-optional.

Reply:
Makes sense to me, I've updated the API to only have `getConfig` and `updateConfig`.